src/ralph/domains/models/domains.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+
+from django.db import models
+from django.conf import settings
+from django.utils.translation import ugettext_lazy as _
+from dj.choices import Choices
+
+from ralph.lib.mixins.models import NamedMixin, TimeStampMixin
+from ralph.lib.permissions import PermByFieldMixin
+
+from ralph.assets.models.assets import (
+    BusinessSegment,
+)
+
+from ralph.assets.models.assets import (
+    Service,
+)
+
+
+class DomainRegistrant(
+    PermByFieldMixin,
+    NamedMixin,
+    TimeStampMixin,
+    models.Model,
+):
+    pass
+
+
+class DomainStatus(Choices):
+    _ = Choices.Choice
+
+    active = _('Active')
+    pending_lapse = _('Pending lapse')
+    pending_transfer = _('Pending transfer away')
+    lapsed = _('Lapsed (inactive)')
+    transfered_away = _('Transfered away')
+
+
+class DomainContract(
+    PermByFieldMixin,
+    TimeStampMixin,
+    models.Model,
+):
+    # we want to store real Domain object in separate macroservice
+    # "django_powerdns_dnssec"
+    domain_name = models.CharField(
+        verbose_name=_('Domain name'),
+        help_text=_('Full domain name'),
+        unique=True,
+        max_length=255
+    )
+    domain_status = models.PositiveIntegerField(
+        choices=DomainStatus(),
+        default=DomainStatus.active.id,
+    )
+    service = models.ManyToManyField(Service)
```

Comment:
i don't know if there are any plans to charge for domains in scrooge, but if there will be such in future then it'll be impossible in current model, since scrooge requires both service and environment.